docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class TensorFlowTransformExample
+    {
+        /// <summary>
+        /// Example use of the TensorFlowEstimator in a ML.NET pipeline.
+        /// </summary>
+        public static void TensorFlowScoringSample()
+        {
+            // Download the ResNet 101 model from the location below.
+            // https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/resnet_v2_101.tgz
+            var modelLocation = @"resnet_v2_101/resnet_v2_101_299_frozen.pb";
+
+            var mlContext = new MLContext();
+            var data = GetTensorData();
+            var idv = mlContext.CreateStreamingDataView(data);
+
+            // Create a ML pipeline.
+            var pipeline = mlContext.Transforms.ScoreTensorFlowModel(modelLocation, new[] { "input" }, new[] { "output" });
+
+            // Run the pipeline and get the transformed values.
+            var estimator = pipeline.Fit(idv);
+            var transformedValues = estimator.Transform(idv);
+
+            // Retrieve model scores.
+            var outScores = transformedValues.AsEnumerable<OutputScores>(mlContext, reuseRowObject: false);
+
+            // Display scores. (for the sake of brevity we display scores of the first 3 classes)
+            foreach (var prediction in outScores)
+            {
+                int numClasses = 0;
+                foreach (var classScore in prediction.output.Take(3))
+                {
+                    Console.WriteLine($"Class #{numClasses++} score = {classScore}");
+                }
+                Console.WriteLine(new string('-', 10));
+            }
+
+            // Results look like below...
+            //Class #0 score = -0.8092947
+            //Class #1 score = -0.3310375
+            //Class #2 score = 0.1119193
+            //----------
+            //Class #0 score = -0.7807726
+            //Class #1 score = -0.2158062
+            //Class #2 score = 0.1153686
+            //----------
+        }
+
+        private const int imageHeight = 224; 
```

Comment:
Not sure how much it matters, I think the "normal" dimensions for the resnet model are 299x299.  I only say that because that's required for the CNTK version and the tensorflow version has 299 in the filename, so that is not very scientific.

Reply:
Looking up the author's github [repo](https://github.com/KaimingHe/deep-residual-networks)  the input shape for ResNet 101 models is  224x224x3

In the CNTL based transform we had in the internal repo also takes input size 224x224  for ResNet 101 models. 

[paper](https://arxiv.org/pdf/1512.03385.pdf)

---
In reply to: [239240041](https://github.com/dotnet/machinelearning/pull/1812#discussion_r239240041) [](ancestors = 239240041)

Reply:
Ah, sorry about that then. Carry on.